prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -27,12 +27,18 @@ def prepare_security_hub_findings(
         if finding.region not in enabled_regions:
             continue
 
-        # Handle status filters, if any
-        if (finding.status != "FAIL" and output_options.send_sh_only_fails) or (
-            output_options.status and finding.status not in output_options.status
-        ):
+        if (
+            finding.status != "FAIL" or finding.muted
+        ) and output_options.send_sh_only_fails:
             continue
 
+        if output_options.status:
+            if finding.status not in output_options.status:
+                continue
+
+            if finding.muted:
+                continue
```

Comment:
Why we are not sending muted findings to Security Hub?

Reply:
This is something I wanted to discuss with you since this has changed from v3 to v4. Currently there is no way not to send muted findings to Security Hub, in v3 we did that since there wasn't a `muted` attribute.

With this, we can send muted findings to Security Hub just if the `status` or `send-sh-only-fails` are not set, what do you think?

Reply:
Sounds good, but I will put something in the Security Hub finding to identify Muted findings.

Reply:
Actually we are sending it as in v3 with the `WARNING` status, since there is nothing more to do there https://github.com/prowler-cloud/prowler/blob/f26f5d3c725b508dd88f376f9cace70072c1b477/prowler/lib/outputs/json_asff/json_asff.py#L18

Security Hub API only supports `PASSED`, `FAILED`, `WARNING` and `NOT_AVAILABLE` per https://docs.aws.amazon.com/securityhub/1.0/APIReference/API_Compliance.html.

Reply:
Perfect then. Thanks!